sdks/java/extensions/sql/build.gradle

```diff
@@ -71,7 +70,15 @@ dependencies {
   compile "com.alibaba:fastjson:1.2.69"
   compile "org.codehaus.janino:janino:3.0.11"
   compile "org.codehaus.janino:commons-compiler:3.0.11"
-  provided "org.checkerframework:checker-qual:3.4.1"
+  compile library.java.jackson_core
+  compile library.java.jsr305
+  compile library.java.vendored_guava_26_0_jre
+  compile "org.mongodb:mongo-java-driver:3.9.1"
+  compile "org.slf4j:slf4j-api:1.7.30"
+  compile "org.apache.beam:beam-vendor-guava-26_0-jre:0.1"
```

Comment:
You already have `library.vendored_guava_26_0_jre` below. That should be the only one.